ci/jenkins/test-vmc.sh

```diff
@@ -40,6 +40,10 @@ DOCKER_REGISTRY=""
 # TODO: change to "control-plane" when testbeds are updated to K8s v1.20
 CONTROL_PLANE_NODE_ROLE="master"
 
+SCP_ANTREA_CI_KEY="scp -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i ${GIT_CHECKOUT_DIR}/jenkins/key/antrea-ci-key"
```

Comment:
why is its name different from the below?

Reply:
Updated. Thanks for reminding.